sources/api/simplefin.py

```
@@ -0,0 +1,117 @@
+            if access_url:
+                return access_url
+
+        access_url = self._claim_access_url(setup_token)
+        with open(ACCESS_URL_FILE, "w") as f:
+            f.write(access_url)
+        os.chmod(ACCESS_URL_FILE, stat.S_IRUSR | stat.S_IWUSR)
+        return access_url
+
+    def _claim_access_url(self, setup_token: str) -> str:
+        claim_url = base64.b64decode(setup_token).decode()
+        response = requests.post(claim_url)
```

Ruff (0.15.17)

[error] Probable use of `requests` call without timeout

(S113)